src/index.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable func-names */
+// import css from "./style.css";
+// declarations
+const form = document.querySelector('#itemForm'); // select the form
+const itemInput = document.querySelector('#itemInput'); // select the input box from the form
+const itemList = document.querySelector('.item-list');
+const inform = document.querySelector('.inform');
+const clearButton = document.querySelector('#clear-list');
+
+let todoItems = [];
```

Comment:
- [ ] **In this milestone of the project, you are required to create an array containing simple to-do tasks. Please implement it.**

You have to create an array of some simple to-do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number]. However, you are free to have more properties or name them differently. 

Please make sure to dynamically append the predefined tasks into DOM.